src/sql/base/browser/ui/colorbox/media/colorbox.css

```diff
@@ -0,0 +1,15 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.colorbox {
+	background: var(--colorbox-bg);
```

Comment:
don't set colors in css, it should be programmic.

Reply:
Curious to know why? Is it just a codestyle for ADS or some other underlying reason?

Reply:
colors should be theme aware, and using variables is just unnecessary and make the code harder to follow.

Reply:
Thanks that makes a lot of sense!